inspire_dojson/cds/rules.py

```diff
@@ -340,6 +340,8 @@ def keywords(self, key, value):
 def accelerator_experiments(self, key, value):
     accelerator = value.get('a')
     experiment = value.get('e')
+    if accelerator == 'Not applicable' or experiment == 'Not applicable':
+        return None
```

Comment:
this is wrong, it might happen that only one is `Not applicable` but not both, e.g. http://cds.cern.ch/record/2320495/export/xm. Better to define a function `ignore_not_applicable` and use that:
```python
def ignore_not_applicable(text):
    return text if text.lower() != 'not applicable' else None
```